pandas/tests/tseries/test_frequencies.py

```diff
@@ -509,6 +509,35 @@ def test_get_freq_code(self):
         assert (frequencies.get_freq_code(offsets.Week(-2, weekday=4)) ==
                 (frequencies.get_freq('W-FRI'), -2))
 
+    def test_frequency_misc(self):
+        assert (frequencies.get_freq_group('T') ==
+                frequencies.FreqGroup.FR_MIN)
+
+        code, stride = frequencies.get_freq_code(offsets.Hour())
+        assert code == frequencies.FreqGroup.FR_HR
+
+        code, stride = frequencies.get_freq_code((5, 'T'))
+        assert code == frequencies.FreqGroup.FR_MIN
+        assert stride == 5
+
+        offset = offsets.Hour()
+        result = frequencies.to_offset(offset)
+        assert result == offset
+
+        result = frequencies.to_offset((5, 'T'))
+        expected = offsets.Minute(5)
+        assert result == expected
+
+        pytest.raises(ValueError, frequencies.get_freq_code, (5, 'baz'))
+
+        pytest.raises(ValueError, frequencies.to_offset, '100foo')
+
+        pytest.raises(ValueError, frequencies.to_offset, ('', ''))
```

Comment:
What are the error messages when you make these method calls?  I would consider using `tm.assert_raises_regex` so that we can check the error message if necessary.

Reply:
No idea.  This is just cut/paste from test_timestamp.

Reply:
Gotcha.  Run these lines of code and see what error messages you get.  Then we can see whether we should check the message in the test.

Reply:
Done, pushing updated...